src/api/vote.ts

```diff
@@ -88,12 +91,7 @@ export abstract class VoteAPI {
     return axios
       .post<IVoteSubmitResponse>(url + VoteAPIMethods.VOTE, JSON.stringify({ txPayload: payload }))
       .then((response) => response.data)
-      .catch((error) => {
-        if (axios.isAxiosError(error)) {
-          throw new Error('Request error: ' + error.message);
-        }
-        throw error;
-      });
+      .catch((error) => this.isApiError(error));
```

Comment:
Same, you can simplify this by just passing `this.isApiError` to the catch (with no args nor parens).